src/main/java/domain/WinningNumber.java

```diff
@@ -0,0 +1,28 @@
+package domain;
+
+public class WinningNumber {
+    private static final String ERROR_DUPLICATED_NUMBER = "[ERROR] 보너스 볼은 당첨 번호와 중복될 수 없습니다.";
+
+    private final LottoTicket winningNumbers;
+    private final LottoNumber bonusBall;
+
+    public WinningNumber(LottoTicket winningNumbers, LottoNumber bonusBall) {
+        this.winningNumbers = winningNumbers;
+        validateBonusBall(bonusBall);
+        this.bonusBall = bonusBall;
```

Comment:
```suggestion
        validateBonusBall(bonusBall);
        this.winningNumbers = winningNumbers;
        this.bonusBall = bonusBall;
```
큰 의미는 없지만, 이 순서가 더 안정감 있게 느껴져요 🙄

Reply:
[`6caf39d`](https://github.com/woowacourse/java-lotto/pull/248/commits/6caf39d3c47bb7fb20b27105c455df64b077e07b) 수정했습니다!